docs/_docs/step-by-step/09-collections.md

```diff
@@ -118,6 +118,7 @@ Add the link to the `staff.html` page:
 ```html
 ---
 layout: default
+title: Staff
```

Comment:
```suggestion

```
There is no need for this front matter setting in this particular context because:
- The page has hard-coded  the title markup (in line 122/123: `<h1>Staff</h1>`
- There is no use of `{{ page.title }}` anywhere